### PR TITLE
fix: update pingcap.json

### PR DESCRIPTION
**Pull request by xuechunL**

### CODE REVIEW

File: configs/pingcap.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   "start_urls": [
-    "https://pingcap.com/docs",
+    "https://pingcap.com/docs/",
+    "https://pingcap.com/docs-cn/",
     "https://pingcap.com/"
   ],
   "stop_urls": [
-    "^((?!https://pingcap.com/doc).)*$"
+    "^((?!https://pingcap.com/doc).)*$",
+    "^(https://pingcap.com/doc-)*$",
```
Comment (s-pace) [inline configs/pingcap.json]: Not a valid json because of this extra coma

File: configs/pingcap.json [hunk 2]
```diff
@@ -1,11 +1,13 @@
 {
   "index_name": "pingcap",
   "start_urls": [
-    "https://pingcap.com/docs",
+    "https://pingcap.com/docs/",
```
Comment (s-pace) [inline configs/pingcap.json]: Already included  from https://pingcap.com/

File: configs/pingcap.json (hunk 1 above)
Comment (xuechunL) [reply configs/pingcap.json]: Thank, I'll fix it

**Comment by xuechunL** [comment]
PTAL @s-pace

**Comment by s-pace** [comment]
LGTM

**Comment by s-pace** [comment]
image
FYI

**Comment by s-pace** [comment]
image
Nothing indexed since nothing available

**Comment by xuechunL** [comment]
@s-pace  we have already update our site, please reopen this PR and merge it.
Thanks a lot!

**Pull request merged**